applications/PFEM2Application/custom_strategies/fracstep_GLS_strategy.h

```diff
@@ -197,17 +197,19 @@ namespace Kratos
 	  typedef Scheme< TSparseSpace, TDenseSpace > SchemeType;
 	  typename SchemeType::Pointer pscheme = typename SchemeType::Pointer(new ResidualBasedIncrementalUpdateStaticScheme< TSparseSpace, TDenseSpace > ());
 
-	  BuilderSolverTypePointer vel_build = BuilderSolverTypePointer(new ResidualBasedEliminationBuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver > (pNewVelocityLinearSolver));
+	  // BuilderSolverTypePointer vel_build = BuilderSolverTypePointer(new ResidualBasedEliminationBuilderAndSolver<TSparseSpace, TDenseSpace, TLinearSolver > (pNewVelocityLinearSolver));
```

Comment:
```suggestion
```

I recommend to remove commented lines

Reply:
Hello @miguelmaso,
thanks for the review! This part is (the two strategies) are a bit tricky. Therefore, for now, let's keep the comments because I'll reinspect them throughout the week.  The main concern of this PR was the last changes in Elements, Conditions and Schemes in the core.
Regards,
Deniz